tokens/governance/treasury-spend.md

```diff
@@ -1,97 +1,103 @@
+
+A member of the Treasury Council must first open [Polkadot.js Apps](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fwss.api.moonbeam.network#/explorer){target=\_blank}  and navigate to the **Extrinsics** tab. From there, the treasury council member will submit a `treasury.spend` call. This call requires specifying the amount (for example, 1,000 GLMR) and the beneficiary account to receive the funds. Once this extrinsic is submitted, it creates a new Treasury Council collective proposal containing the specified spending information. Other members of the treasury council vote on the proposal with the `treasuryCouncilCollective.vote` extrinsic by providing a hash to the proposal, the proposal index, and aye/nay boolean to approve. Finally, a member of the treasury council closes the vote with the `treasuryCouncilCollective.close` extrinsic.
+ 
+!!! note
+    There is no on-chain action for the proposer or beneficiary of the treasury spend request.
+    All treasury spend actions will be completed by members of the treasury council.
 
-1. Select the account that will be the creator of the Treasury proposal. Make sure that the account selected here is sufficiently funded with your proposal bond
-2. Select the beneficiary of the proposal. This should correspond to the account indicated in your forum post
-3. Indicate the value of the spend proposal. The required bond is {{ networks.moonbeam.treasury.proposal_bond}}% of the proposal's value. The bond committed to a Treasury spend proposal is irrevocable
-4. Review each field for accuracy then press **Submit proposal** and sign the transaction
+Note that this process has changed significantly from prior treasury processes, where tokenholders could submit treasury proposals with bonds attached. Now, no on-chain action is necessary to receive a treasury proposal - rather, all that is needed is to raise a treasury council request on the [Moonbeam Forum](https://forum.moonbeam.network/c/governance/treasury-proposals/8){target=\_blank} and the treasury council will take care of the on-chain component to process a treasury spend if the proposal is approved.
 
-![Submitting a Treasury spend proposal](/images/tokens/governance/treasury-proposals/treasury-proposal-3.webp)
+## After the Treasury Council Approval
 
-After submitting your Treasury proposal, you can refresh Polkadot.js Apps and you should see your proposal listed. If you don't see your proposal appear here, your account may not have contained sufficient funds for the required bond.
+The final step in the treasury spend request flow is for the `treasury.payout` extrinsic to be called. This may be called by anyone after the treasury council has approved the spend on-chain by providing the index of the treasury spend approved. Most likely, this step will be completed by a member of the treasury council, however, this step is documented here as it is one of the only steps that anyone can complete. 
```

Comment:
This is copied as well.

Maybe what this guide needs is like a "Next Steps" section, instead of the "Treasury Council Voting Process" and "After the Treasury Council Approval" sections, which _briefly_ outlines the remainder of the steps, links to the Learn page where it makes sense so users can learn more, then lastly adds the steps to execute the treasury payout